store/tracekv/README.md

```diff
@@ -0,0 +1,15 @@
+# tracekv
```

Comment:
is this related to state streaming? it seems they overlap in use cases?

Reply:
Nah, it's not. In store v1 there was `listenkv` (no longer exists) and `tracekv` (ported to v2). `tracekv` is useful for directly plugging into an `io.writer` to evaluate R/W on stores.

I think state streaming is more sophisticated and heavy. There you're piping state changes to some external sink.